DATA_SOURCES.md

```diff
@@ -294,7 +294,7 @@ For many European countries, data is available from [ENTSO-E](https://transparen
   - Other renewables: [IRENA](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Mar/IRENA_RE_Capacity_Statistics_2020.pdf)
   - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Nicaragua: [Climatescope](http://global-climatescope.org/en/country/nicaragua/)
-- Nigeria: [SO Grid](https://www.niggrid.org)
+- Nigeria: [USAID](https://www.usaid.gov/powerafrica/nigeria)
```

Comment:
Change the Nigeria reference to have 2 bullet points. 

* One for the Data source: SO Grid
* One for the Capacity: USAID